go.mod

```diff
@@ -10,10 +10,10 @@ require (
 	github.com/coreos/ignition/v2 v2.9.0
 	github.com/ghodss/yaml v1.0.1-0.20190212211648-25d852aebe32
 	github.com/google/go-cmp v0.5.6
-	github.com/jaypipes/ghw v0.7.1-0.20210419135914-b8b1e31b27f5
+	github.com/jaypipes/ghw v0.8.1-0.20210605191321-eb162add542b
```

Comment:
next time, please split updates of go.mod, go.sum and vendor in its own separate commit